File: C/24BCSH93/1st Semester/assignment6/question3.c
#include <stdio.h>

int main() {
    int num, i, j;
    printf("Enter the number: ");
    scanf("%d", &num);

    for (i = 1; i <= num * 2; i++) {
        for (j = 1; j <= i; j++) {
            if (i % 2 != 0) {
                printf("%d ", j);
            }
        }
        if (i % 2 != 0) {
            printf("\n");
        }
    }
    return 0;
}

File: MATLAB/assignment11/question3.m
%Q3. Find the greatest number among three unequal numbers.

a = input('Enter first number: ');
b = input('Enter second number: ');
c = input('Enter third number: ');

if a > b && a > c
    disp(['Greatest is: ', num2str(a)])
elseif b > c
    disp(['Greatest is: ', num2str(b)])
else
    disp(['Greatest is: ', num2str(c)])
end
